composer.json

```diff
@@ -31,7 +31,7 @@
         "laminas/laminas-zendframework-bridge": "^1.0"
     },
     "require-dev": {
-        "doctrine/migrations": "^1.0 || ^2.0",
+        "doctrine/migrations": "^1.0 || ^2.0 || ^3.0",
```

Comment:
Do we still need to support `^1`? At this point, it looks like it's really old (>2y, and `^2` is easy to migrate to), and we'll probably never get to test `^2` if our CI does a `--prefer-lowest` installation